pkg/ccl/changefeedccl/cdceval/constraint.go

```diff
@@ -102,8 +102,5 @@ func constrainSpansBySelectClause(
 }
 
 func schemaTS(execCtx sql.JobExecContext) hlc.Timestamp {
-	if execCtx.Txn() != nil {
-		return execCtx.Txn().ReadTimestamp()
-	}
```

Comment:
@cockroachdb/cdc-prs This bit is worth looking at.

Reply:
Thanks for the heads up. I think this is fine since we cache the result anyway, and we just need consistent results while validating, not ones specific to the transaction.

Reply:
Cool. And, I _believe_ that in the context this was being called, that context would have been nil all of the time anyway. So hopefully this is a noop.